components/BottomDrawer/src/MDCBottomDrawerTransitionController.h

```diff
@@ -0,0 +1,27 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+// TODO: (larche) Cite spec article.
+/**
+ A controller for scrubbable scrolling (aka dragging) vertical pan gesture-based modal transitions
+ from the bottom of the screen (initiated, perhaps, in an MDCBottomAppBarView).
+ */
+@interface MDCBottomDrawerTransitionController: UIPercentDrivenInteractiveTransition
```

Comment:
Naming is hard: `MDCBottomDrawerScrubbableTransition` ? Ending with *Controller makes it feel like a VC.